src/main/java/org/opentripplanner/middleware/controllers/api/OtpUserController.java

```diff
@@ -49,4 +81,18 @@ boolean preDeleteHook(OtpUser user, Request req) {
         }
         return true;
     }
+
+    /**
+     * HTTP endpoint to get the {@link OtpUser} entity, if it exists, from an {@link Auth0UserProfile} attribute
+     * available from a {@link Request} (this is the case for '/api/secure/' endpoints).
+     */
+    private OtpUser getUserFromRequest(Request req, Response res) {
+        Auth0UserProfile profile = Auth0Connection.getUserFromRequest(req);
+        OtpUser user = profile.otpUser;
+
+        if (user == null) {
+            logMessageAndHalt(req, HttpStatus.NOT_FOUND_404, String.format(NO_USER_WITH_AUTH0_ID_MESSAGE, profile.auth0UserId), null);
```

Comment:
This should return an HTTP 403 error since it seems that whatever request this is will fail if an authenticated user is not present in the request.